play.py

```diff
@@ -45,16 +67,17 @@ def do_GET(self):
         output = [
             int(joystick[0] * 80),
             int(joystick[1] * 80),
-            int(round(joystick[2])),
-            int(round(joystick[3])),
-            int(round(joystick[4])),
+            int(joystick[2]),
+            int(joystick[3]),
+            int(joystick[4]),
+            #int(round(joystick[5])),
```

Comment:
Ok, regarding the comment. Also, the comma after the last element was in your original code. Was that a typo, or does it mean something?